example/src/Examples.tsx

```diff
@@ -336,6 +437,12 @@ const styles = StyleSheet.create({
 });
 
 export const examples: Props[] = [
+  {
+    title: 'Custom step marker settings',
+    render() {
+      return <SliderExampleWithCustomMarker />;
```

Comment:
I liked the order of having more complex examples at the end of the list.
I'm happy to see additional example, always good to have as many, but can we move this one to where other "Custom step marker" examples are? 🙏